spec/factories/collection_types.rb

```diff
@@ -1,7 +1,8 @@
 FactoryGirl.define do
   factory :collection_type, class: Hyrax::CollectionType do
-    sequence(:id) { |n| n }
-    sequence(:title) { |n| "Title #{n}" }
+    sequence(:id) { |n| format("%010d", n + 2) }
```

Comment:
Duh, this is not needed because the special types will be created in the dev db of the test app and not the test db.  I took this out and tests pass.